src/platform/forms-system/src/js/components/FormNav.jsx

```diff
@@ -10,72 +9,96 @@ import {
-    // (the chapter index will probably be wrong, but this isn’t a scenario that happens in normal use)
-    if (!page) {
-      page = formPages.find(
-        p => `${formConfig.urlPrefix}${p.path}` === currentPath,
-      );
+  let current;
+  let chapterName;
+  if (page) {
+    current = chapters.indexOf(page.chapterKey) + 1;
+    // The review page is always part of our forms, but isn’t listed in chapter list
+    chapterName =
+      page.chapterKey === 'review'
+        ? formConfig?.customText?.reviewPageTitle || REVIEW_APP_DEFAULT_MESSAGE
+        : formConfig.chapters[page.chapterKey].title;
+    if (typeof chapterName === 'function') {
+      chapterName = chapterName();
     }
+  }
+
+  const stepText = `Step ${current} of ${chapters.length}: ${chapterName}`;
+  const showHeader = Math.abs(current - index) === 1;
 
-    let current;
-    let chapterName;
-    if (page) {
-      current = chapters.indexOf(page.chapterKey) + 1;
-      // The review page is always part of our forms, but isn’t listed in chapter list
-      chapterName =
-        page.chapterKey === 'review'
-          ? formConfig?.customText?.reviewPageTitle ||
-            REVIEW_APP_DEFAULT_MESSAGE
-          : formConfig.chapters[page.chapterKey].title;
-      if (typeof chapterName === 'function') {
-        chapterName = chapterName();
+  // The goal with this is to quickly "remove" the header from the DOM, and
+  // immediately re-render the component with the header included. This should
+  // ensure that VoiceOver on iOS will pick up on the new <h2>
+  useEffect(
+    () => {
+      if (current > index + 1) {
+        setIndex(index + 1);
+      } else if (current === index) {
+        setIndex(index - 1);
       }
-    }
 
-    const stepText = `Step ${current} of ${chapters.length}: ${chapterName}`;
-
-    return (
-      <div>
-        <SegmentedProgressBar total={chapters.length} current={current} />
-        <div className="schemaform-chapter-progress">
-          <div
-            aria-valuenow={current}
-            aria-valuemin="1"
-            aria-valuetext={stepText}
-            aria-valuemax={chapters.length}
-            className="nav-header nav-header-schemaform"
-          >
-            <h2 className="vads-u-font-size--h4">{stepText}</h2>
-          </div>
+      return () => {
+        focusElement('.nav-header > h2');
+      };
+    },
+    [current, index],
+  );
+
+  return (
+    <div>
+      <SegmentedProgressBar total={chapters.length} current={current} />
+      <div className="schemaform-chapter-progress">
+        <div
+          aria-valuenow={current}
+          aria-valuemin="1"
+          aria-valuetext={stepText}
+          aria-valuemax={chapters.length}
+          className="nav-header nav-header-schemaform"
+        >
+          {showHeader && (
+            <h2 id="nav-form-header" className="vads-u-font-size--h4">
+              {stepText}
+            </h2>
```

Comment:
State management determines whether  or not we render this, so that we can do a quick re-render with it included so that VoiceOver will be aware of the change in `stepText`